tests/tests/test-balance.ts

```diff
@@ -6,7 +6,7 @@ import { createAndFinalizeBlock, describeWithMoonbeam, customRequest } from "./u
 
 describeWithMoonbeam("Moonbeam RPC (Balance)", `simple-specs.json`, (context) => {
   const GENESIS_ACCOUNT = "0x6be02d1d3665660d22ff9624b7be0551ee1ac91b";
-  const GENESIS_ACCOUNT_BALANCE = "340282366920938463463374607431768211455";
+  const GENESIS_ACCOUNT_BALANCE = "340282366920938463463374607431768111455";
```

Comment:
Why and how was the balance changed?

Reply:
Amar mentioned it (for got where) but it has changed because part of the balance has been "reserved" for the staking (100_000)

Reply:
good to know thanks